### PR TITLE
docs(ajax): Adding more examples and cases of how ajax operator can be used.

### PR DESCRIPTION
**Pull request by paschalidi**
👋 loving how great job you people doing here! Thanks for giving us the opportunity to use this library.

**Description:**
I have the feeling that since `ajax` is one of the useful operators we have in Rxjs for making requests it would be nice if the documentation have more examples. Therefore I opened this PR that will add few examples on Stackblitz.
I tried to follow best your style but since its my first PR here I would be happy to hear your feedback. I am positive that I am gonna work on that in order to have this documented well for future referenced.

**Related issue:**
https://github.com/ReactiveX/rxjs/issues/4652

### CODE REVIEW

File: src/internal/observable/dom/ajax.ts [hunk 1]
```diff
@@ -5,15 +5,82 @@ import {  AjaxObservable, AjaxCreationMethod  } from './AjaxObservable';
...
+ *
+ * const obs$ = ajax(`https://api.github.com/users?per_page=5`).pipe(
+ *   map(userResponse => console.log('users: ', userResponse)),
+ *   catchError(error => {
+ *     console.log('error: ', error)
+ *     return of(error)
+ *   })
+ * );
+ *
+ * const subscribe = obs$.subscribe();
```
Comment (benlesh) [inline src/internal/observable/dom/ajax.ts]: We're not doing anything with the emitted values in any of these.

File: src/internal/observable/dom/ajax.ts [hunk 2]
```diff
@@ -5,15 +5,82 @@ import {  AjaxObservable, AjaxCreationMethod  } from './AjaxObservable';
...
+ * ## Using ajax() to fetch the response object that is being returned from API.
+ * ```javascript
+ * import { ajax } from 'rxjs/ajax';
+ * import { map, catchError } from 'rxjs/operators';
+ * import { of } from 'rxjs';
+ *
+ * const obs$ = ajax(`https://api.github.com/users?per_page=5`).pipe(
+ *   map(userResponse => console.log('users: ', userResponse)),
+ *   catchError(error => {
+ *     console.log('error: ', error)
```
Comment (benlesh) [inline src/internal/observable/dom/ajax.ts]: missing semicolons throughout. Please double check. :)

File: src/internal/observable/dom/ajax.ts (hunk 1 above)
Comment (paschalidi) [reply src/internal/observable/dom/ajax.ts]: Hey @benlesh thanks for the very quick review. Yes I see but when you open Stackblitz then consoles dont really do anything if you dont subscribe, right?

**Comment by paschalidi** [comment]
@benlesh please have another look when you get the time.

**Comment by paschalidi** [comment]
closing was result of missclick

**Comment by paschalidi** [comment]
@benlesh thanks for the nice input you gave! please let me know what holds you back here.

**Comment by paschalidi** [comment]
Sure @dzhavat happy to help.

**Comment by paschalidi** [comment]
@dzhavat does it make sense that I do that for all the documentation rxjs has in a different PR?

**Comment by dzhavat** [comment]
@paschalidi It's probably a good idea but there's already a PR (#4635) doing that. I'm not sure if it covers all documentation but I'd guess so.

**Comment by paschalidi** [comment]
@dzhavat how are things over here?

**Comment by dzhavat** [comment]
@paschalidi Thanks for making the changes. Like I mentioned in my first comment, I'm just a fellow contributor. The final approval and merge of this PR doesn't depend on me :)

**Comment by paschalidi** [comment]
@benlesh sir anything else I could do for this PR to get better?

**Pull request merged**